compare.py

```diff
@@ -9,7 +9,9 @@ def __init__(self, renewals_input_path):
         for i in open(renewals_input_path):
             renewal = Renewal(**json.loads(i))
             regnum = renewal.regnum
-            if not isinstance(regnum, list):
+            if regnum is None:
+                regnum = ()
```

Comment:
Seems like it would make more sense to massage it to an empty list instead of an empty tuple:
```
regnum = []
```
since that would fit into what the code is doing here for single values.